### ISSUE TITLE
Implement Properties Route

### ISSUE THREAD

**Issue by tobiasschweizer** [dhlab-basel/Knora#134]
I started to implement the properties route.

The answer should look like this:

```json
{
    "status": 0,
    "properties": {
        "salsah:comment": {
            "label": "Comment",
            "valuetype_id": "1",
            "valuetype": "textval",
            "guielement": "textarea",
            "attributes": "width=95%;rows=7;wrap=soft",
            "is_annotation": "1",
            "pid": "20",
            "values": [

                {
                    "person_id": "24",
                    "comment": null,
                    "textval": "Defekter Druckstock",
                    "val": "Defekter Druckstock",
                    "id": "49073",
                    "lastmod": "2012-10-07 17:56:18",
                    "lastmod_utc": "2012-10-07 15:56:18"
                }
            ]

        }
    }
}
```

**Comment by tobiasschweizer** [comment]
However, this is not the format returned by `getResourceProperties` since this method returns a Sequence of `PropertyV1`.

@geer: Do you know if we have handled this format before?

**Comment by benjamingeer** [comment]
It doesn't look familiar. :( But maybe with a little transformation, you can generate it from the `Seq[PropertyV1]`.

**Comment by Geer** [comment]
Hey guys, good morning I'm little sure that someone of you want to tag some
other Geer.

Cheers and hope that you find your answer

**Comment by tobiasschweizer** [comment]
Wow there are two geers :-)

**Comment by tobiasschweizer** [comment]
> It doesn't look familiar. :( But maybe with a little transformation, you can generate it from the Seq[PropertyV1].

The only thing I recognize is the typing of the values (textval) as in `convertCreateValueResponseV1ToResourceCreateValueResponseV1` in `MessageUtil`

**Comment by tobiasschweizer** [comment]
https://github.com/dhlab-basel/Knora/commit/434e4d2f025f463aa208a140f3b5ac2d15860e43

There needs still refactoring to be done to turn PropertyV1 into the desired format

**Comment by tobiasschweizer** [comment]
I made progress on regions. I added a query for regions to the context query (https://github.com/dhlab-basel/Knora/commit/23d8c74b57868a1beda1b078c3ccea558c9cfa05).

And I saw that the format the format for `resinfo.regions` is the same as is required for the properties route. So once I managed to turn PropertyV1 into that format, I can reuse this method.

I also discovered another thing: there is a parameter "regnum" for the `resources` route.
